### PR TITLE
release server manager v1.12.0

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: manager/latest-linux.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+version: 1.12.0
 files:
-  - url: https://s3.amazonaws.com/outline-releases/manager/linux/1.11.0/1/Outline-Manager.AppImage
-    sha512: 2vMpB1tCG6PkpyLcLgCOKc/+vi9b4Gn8DYTebmTZ7lT73+QffHNwnNuj1EwD3GGaUmhzEzYJXDy4mV1kzuN7kQ==
-    size: 106854835
-    blockMapSize: 113095
-path: https://s3.amazonaws.com/outline-releases/manager/linux/1.11.0/1/Outline-Manager.AppImage
-sha512: 2vMpB1tCG6PkpyLcLgCOKc/+vi9b4Gn8DYTebmTZ7lT73+QffHNwnNuj1EwD3GGaUmhzEzYJXDy4mV1kzuN7kQ==
-releaseDate: '2022-09-13T21:20:29.762Z'
+  - url: Outline-Manager.AppImage
```
Comment (jyyi1) [inline manager/latest-linux.yml:3]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/linux/1.12.0/1/Outline-Manager.AppImage
```

File: manager/latest-linux.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    sha512: YnIxaLfFVoqUwtmay67UrytZiwn2uZBPctIpzbqmkxsyxfPRTDc+aa8g3idEktE15w/PvJ+pEo3vIItzDkI+Ag==
+    size: 106805774
+    blockMapSize: 113186
+path: Outline-Manager.AppImage
```
Comment (jyyi1) [inline manager/latest-linux.yml:7]:
```suggestion
path: https://s3.amazonaws.com/outline-releases/manager/linux/1.12.0/1/Outline-Manager.AppImage
```

File: manager/latest-mac.yml [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
-  - url: https://s3.amazonaws.com/outline-releases/manager/macos/stable/Outline-Manager.dmg
-    sha512: n1rkI8/zaj1WtjkQHpw0i8FYhs+byvXxA97o8RM5pGcTDpEAECjX90E4SVAkXRmG+Yjk9DDOvlcGki1ZEQVs1Q==
-    size: 105613729
-  - url: https://s3.amazonaws.com/outline-releases/manager/macos/stable/Outline-Manager.dmg
-    sha512: n1rkI8/zaj1WtjkQHpw0i8FYhs+byvXxA97o8RM5pGcTDpEAECjX90E4SVAkXRmG+Yjk9DDOvlcGki1ZEQVs1Q==
-    size: 105613729
-path: https://s3.amazonaws.com/outline-releases/manager/macOS/1.11.0/1/Outline-Manager.zip
-sha512: yVahuafroRvkRoUg+FGvkaGeQgRGrRyMI3ti1OU2M311cAOK4l2CbdMtpKoXqAfhoORqmxrqXLx1QJqmGMSfNg==
-releaseDate: '2022-09-13T21:38:10.395Z'
+  - url: Outline-Manager.zip
```
Comment (jyyi1) [inline manager/latest-mac.yml:3]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.zip
```

File: manager/latest-mac.yml [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+    sha512: uq9zndRKzNttDJ2qiRJAt1je8WHdg0uMK+cTar0DnYvSDrYliJ5a1pti7rFVy8bMNKt0hS4Ex21eFJpVOKh7/g==
+    size: 101593750
+  - url: Outline-Manager.dmg
```
Comment (jyyi1) [inline manager/latest-mac.yml:6]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.dmg
```

File: manager/latest-mac.yml [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+    sha512: 6+H0KKWJ27REC8T97Bxaaw03r5RflxSgg4U4xSK2+Lp1YROh78CQRnz3aN8KAfubcPwf+J8x+vy0TD6IW7H8aw==
+    size: 105625369
+  - url: Outline-Manager.dmg
```
Comment (jyyi1) [inline manager/latest-mac.yml:9]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.dmg
```

File: manager/latest-mac.yml [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+    sha512: 6+H0KKWJ27REC8T97Bxaaw03r5RflxSgg4U4xSK2+Lp1YROh78CQRnz3aN8KAfubcPwf+J8x+vy0TD6IW7H8aw==
+    size: 105625369
+  - url: Outline-Manager.dmg
```
Comment (jyyi1) [inline manager/latest-mac.yml:12]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.dmg
```

File: manager/latest.yml [hunk 7]
```diff
@@ -1,8 +1,8 @@
-version: 1.11.0
+version: 1.12.0
 files:
-  - url: https://s3.amazonaws.com/outline-releases/manager/windows/1.11.0/1/Outline-Manager.exe
-    sha512: 4gjifs0q08JX+g3oGm+9TS40Ztd8wqegcPPPlbv8LSE+4nDHXQ4BF9BAChHSrGDJzxANVxuTpxdgsjO/Pyxbdw==
-    size: 76237824
-path: https://s3.amazonaws.com/outline-releases/manager/windows/1.11.0/1/Outline-Manager.exe
-sha512: 4gjifs0q08JX+g3oGm+9TS40Ztd8wqegcPPPlbv8LSE+4nDHXQ4BF9BAChHSrGDJzxANVxuTpxdgsjO/Pyxbdw==
-releaseDate: '2022-09-13T21:27:41.946Z'
+  - url: Outline-Manager.exe
```
Comment (jyyi1) [inline manager/latest.yml:3]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/windows/1.12.0/1/Outline-Manager.exe
```

File: manager/latest.yml [hunk 8]
```diff
@@ -1,8 +1,8 @@
...
+    sha512: yUovaI5HEuPAUXueNQHFH9YJKNq1p2I8rLl1YwyEN6US9Pn/nTWCdKGae33G9ZB3j2X52vtXyJhSFUn+3fr/Yg==
+    size: 76242688
+path: Outline-Manager.exe
```
Comment (jyyi1) [inline manager/latest.yml:6]:
```suggestion
path: https://s3.amazonaws.com/outline-releases/manager/windows/1.12.0/1/Outline-Manager.exe
```

**Review by jyyi1 — CHANGES_REQUESTED** [review]
Please update the file paths to the absolute url.

**Comment by daniellacosse** [comment]
> Please update the file paths to the absolute url.

Ah, I just caught that myself

File: manager/latest-mac.yml [hunk 9]
```diff
@@ -1,17 +1,17 @@
-version: 1.11.0
+version: 1.12.0
 files:
-  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.11.0/1/Outline-Manager.zip
-    sha512: yVahuafroRvkRoUg+FGvkaGeQgRGrRyMI3ti1OU2M311cAOK4l2CbdMtpKoXqAfhoORqmxrqXLx1QJqmGMSfNg==
-    size: 101604427
+  - url: tps://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.zip
```
Comment (jyyi1) [inline manager/latest-mac.yml:3]:
```suggestion
  - url: https://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.zip
```

File: manager/latest-mac.yml [hunk 10]
```diff
@@ -1,17 +1,17 @@
...
+    size: 105625369
   - url: https://s3.amazonaws.com/outline-releases/manager/macos/stable/Outline-Manager.dmg
-    sha512: n1rkI8/zaj1WtjkQHpw0i8FYhs+byvXxA97o8RM5pGcTDpEAECjX90E4SVAkXRmG+Yjk9DDOvlcGki1ZEQVs1Q==
-    size: 105613729
-path: https://s3.amazonaws.com/outline-releases/manager/macOS/1.11.0/1/Outline-Manager.zip
-sha512: yVahuafroRvkRoUg+FGvkaGeQgRGrRyMI3ti1OU2M311cAOK4l2CbdMtpKoXqAfhoORqmxrqXLx1QJqmGMSfNg==
-releaseDate: '2022-09-13T21:38:10.395Z'
+    sha512: 6+H0KKWJ27REC8T97Bxaaw03r5RflxSgg4U4xSK2+Lp1YROh78CQRnz3aN8KAfubcPwf+J8x+vy0TD6IW7H8aw==
+    size: 105625369
+path: tps://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.zip
```
Comment (jyyi1) [inline manager/latest-mac.yml:15]:
```suggestion
path: https://s3.amazonaws.com/outline-releases/manager/macos/1.12.0/1/Outline-Manager.zip
```

**Review by jyyi1 — APPROVED** [review]
LGTM now, tried every link, all downloadable.

**Pull request merged**